docs/docs/sample-scripts/flow/cancel-all-running-flow-runs.md

```diff
@@ -1,3 +1,8 @@
+---
+tags:  
+  - Cleanup

```

Comment:
This doesn't seem like a relevant tag to this script. To me, cleanup is more about removing resources. Cancelling running flows is more like management at scale/automation, which could be said about all scripts.